packages/www/hooks/use-api/endpoints/user.ts

```diff
@@ -227,9 +235,34 @@ export const getBillingUsage = async (
     {}
   );
 
+  if (!usage) {
+    return [
+      res,
+      {
+        TotalUsageMins: 0,
+        DeliveryUsageMins: 0,
+        StorageUsageMins: 0,
+      } as BillingUsageData | ApiError,
+    ];
+  }
+
   return [res, usage as BillingUsageData | ApiError];
 };
 
+export const getUpcomingInvoice = async (
+  stripeCustomerId: string
+): Promise<[Response, any | ApiError]> => {
```

Comment:
Can you add proper typing (no `any` for our own stuff) if/when you have a chance? Seems like there's already an `UpcomingInvoice` type declared?